libpod/options.go

```diff
@@ -844,6 +844,19 @@ func WithPIDNSFrom(nsCtr *Container) CtrCreateOption {
 	}
 }
 
+// WithUserNSKeepID indicates that container should add user entry to passwd
+// file, since the UID will be mapped into the container, via user namespace
+func WithUserNSKeepId() CtrCreateOption {
```

Comment:
Can we rename this? Maybe "WithAddCurrentUserToPasswd" or something more clear about what it does?